src/transaction.js

```diff
@@ -1221,21 +1222,24 @@ Transaction.prototype.summary = function() {
     initialSubmitIndex: this.initialSubmitIndex,
     lastLedgerSequence: this.lastLedgerSequence,
     state: this.state,
-    server: this._server ? this._server._opts.url : undefined,
```

Comment:
Yep, transactions are submitted to all servers now. I think this would always be undefined currently, just removed it instead